scripts/core/editor3/helpers/highlights.js

```diff
@@ -518,6 +518,38 @@ function addCommentsForServer(editorState) {
     return setCustomDataForEditor(editorState, editor3DataKeys.__PUBLIC_API__comments, comments);
 }
 
+/**
+ * @ngdoc method
+ * @name handleBeforeInputHighlights
+ * @description prevents inheriting of highlight styles
+ */
+export function handleBeforeInputHighlights(onChange, chars, editorState) {
```

Comment:
On handleBeforeInput at line 272 (https://github.com/tomaskikutis/superdesk-client-core/blob/e61169293cb6cf5addc8519d1f9960a9f78aef15/scripts/core/editor3/components/Editor3Component.jsx#L272) starts a code that seams to do the same thing like the code on this function. Can you refactor the code from handleBeforeInput  and add a call to this function.